lib/src/upgrade_messages.dart

```diff
@@ -161,7 +161,7 @@ class UpgraderMessages {
       case 'en':
       default:
         message =
-            'A new version of {{appName}} is available! Version {{currentAppStoreVersion}} is now available-you have {{currentInstalledVersion}}.';
+            'A new version of {{appName}} is available! Version {{currentAppStoreVersion}} is now available - you have {{currentInstalledVersion}}.';
```

Comment:
This change is not related to Android and the Play Store.

Reply:
Yes, it was for my project and should not have added it

Reply:
I will change it back to the what you originally did

Reply:
I just made a new commit to reset it to the default

Reply:
@Williano Check again.